### DISCUSSION TITLE
Search and group results

### DISCUSSION THREAD

**Discussion by vongohren** [community/community#142381] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I would like to be able to search for some code, and just group things by my choice, f.ex repository, or files. 

Its helpful to figure out what repositories that have certain search representedn in them. Finding dependencies on a bit more diffuse search than version in dependency file and so on. 

Is this possible?

**Comment by CodewithShahriar** [comment]
Yes, you can search and group code across repositories using the GitHub Search API, tools like Sourcegraph, or by writing a custom script to filter by repository, file, or dependencies.

**Reply by vongohren** [reply]
Thanks for the input, was hoping to be able to do this with the UI. But I see that github has answered below

**Comment by look** [comment, marked as answer]
@vongohren thank you for sharing your use case. This isn't possible right now in the UI, but I'll share the feedback with the rest of the team.

**Reply by vongohren** [reply]
Thanks for sharing!
